packages/oscal-react-library/src/components/OSCALProfile.js

```diff
@@ -17,7 +17,7 @@ import { OSCALAnchorLinkHeader } from "./OSCALAnchorLinkHeader";
  * represents the baseline of selected controls from one or more control catalogs.
  * For more information see: https://pages.nist.gov/OSCAL/concepts/layer/control/profile/
  *
- * @param {Object} props
+ * @param {*} props
```

Comment:
`profile` is not an attribute of `Object` and I am _not_ diving into typing `OSCALProfile` right now lol